### PR TITLE
ADEN-6000 Fixed Evolve tests after enabling PAL in US

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestDfpParamsPresent.java#L6](https://github.com/Wikia/selenium-tests/blob/7c6fe3950aa31908e8a984abeb3dd0277231f6b4/src/test/java/com/wikia/webdriver/testcases/adstests/TestDfpParamsPresent.java#L6): Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsEvolveObject'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestDfpParamsPresent.java [hunk 1]
```diff
@@ -53,23 +53,4 @@ public void dfpParamsPresentOasis(String wikiName,
     ads.verifyGptIframe(adUnit, slot, "gpt");
     ads.verifyGptParams(slot, pageParams, slotParams);
   }
-
-  @Test(
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "dfpEvolveParamsOasis",
-      groups = {"Ads", "AdsEvolveOasis"}
-  )
-  public void dfpEvolveParamsPresentOasis(String wikiName,
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestDfpParamsPresent.java]: https://github.com/Wikia/app/blob/f78dded46b91a891cdd89b3ae1072a8a7f501646/extensions/wikia/AdEngine/js/spec/provider/evolve2.spec.js

**Comment by dianafa** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestDfpParamsPresent.java#L6](https://github.com/Wikia/selenium-tests/blob/9eb2e59c76b3659a305bfd02d45cff03023c2802/src/test/java/com/wikia/webdriver/testcases/adstests/TestDfpParamsPresent.java#L6): Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsEvolveObject'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestDfpParamsPresent.java#L6](https://github.com/Wikia/selenium-tests/blob/666b953548f8d1eca0b387a4bf2afb0d1836b1f6/src/test/java/com/wikia/webdriver/testcases/adstests/TestDfpParamsPresent.java#L6): Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsEvolveObject'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**